src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -604,17 +636,13 @@ ucs_status_t uct_rc_mlx5_init_rx_tm(uct_rc_mlx5_iface_common_t *iface,
     srq_init_attr->cq                  = iface->super.super.cq[UCT_IB_DIR_RX];
     srq_init_attr->tm_cap.max_num_tags = iface->tm.num_tags;
 
-    /* 2 ops for each tag (ADD + DEL) and extra ops for SYNC.
-     * There can be up to "max_cancel_sync_ops" SYNC ops during cancellation.
-     * Also we assume that there can be up to two pending SYNC ops during
-     * unexpected messages flow. */
-    iface->tm.cmd_qp_len = (2 * iface->tm.num_tags) + max_cancel_sync_ops + 2;
+    iface->tm.cmd_qp_len = (2 * iface->tm.num_tags) + 2;
```

Comment:
I'd leave the first comment line  `2 ops for each tag (ADD + DEL) and extra ops for SYNC`